x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -1,9 +1,101 @@
+appender.deprecated_audit_rolling.type = RollingFile
+appender.deprecated_audit_rolling.name = deprecated_audit_rolling
+appender.deprecated_audit_rolling.fileName = ${sys:es.logs.base_path}${sys:file.separator}${sys:es.logs.cluster_name}_access.log
```

Comment:
@bizybot See the discussion starting here https://github.com/elastic/elasticsearch/issues/31046#issuecomment-413783224

Basically, Filebeat has to support whatever format we're defaulting too. So if we're not enabling the new appender in 6.x Filebeat will not use this in 6.x, so we might as well not port this to 6.x because Filebeat is the important reason we do this (sort of, the root reason is to get rid of the `IndexAuditTrail`, but we need Filebeat to consume or logs before we can do that).

And since we ❤️ our users, we don't want to break any systems they have set up right now that are consuming the present format. So, the compromise is to enable both of them.